src/resources/views/list.blade.php

```diff
@@ -21,6 +22,7 @@
                             <option value="role">{{ trans('teamspeak::seat.role_filter') }}</option>
                             <option value="corporation">{{ trans('teamspeak::seat.corporation_filter') }}</option>
                             <option value="alliance">{{ trans('teamspeak::seat.alliance_filter') }}</option>
+                            <option value="title">{{ trans('teamspeak::seat.title_filter') }}</option>
```

Comment:
best to use `csrf_field` helper, so we don't have to post an hotfix if framework change something in the way csrf is handled.